components/blitz/src/ome/formats/importer/transfers/AbstractFileTransfer.java

```diff
@@ -159,7 +160,7 @@ protected void printLine() {
     }
 
     /**
-     * Method used by subclasses during {@link #afterTransfer(int, List<String>)}
+     * Method used by subclasses during {@link ome.formats.importer.transfers.FileTransfer#afterTransfer(int, List)}
```

Comment:
I think you're right. I was puzzled over this myself. (Sometimes after some changes Eclipse gets confused and needs a kick.)